packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -782,9 +796,42 @@ const preparedCharts = computed(() => {
 			),
 			annotations
 		);
-
-		charts[variable].layer.push(...createInterventionChartMarkers(groupedInterventionOutputs.value[variable]));
```

Comment:
Why do we delete this line? Do we not show the variable intervention on the variable chart anymore?

Reply:
I suppose we could leave it even though it's duplicated ion the intervention chart.